examples/pcc-transport-chatbot/client-react/src/providers/RTVIProvider.tsx

```diff
@@ -0,0 +1,38 @@
+'use client';
+
+import { RTVIClient } from '@pipecat-ai/client-js';
+import { DailyTransport } from '@pipecat-ai/daily-transport';
+import { RTVIClientProvider } from '@pipecat-ai/client-react';
+import { PropsWithChildren, useEffect, useState } from 'react';
+
+const MY_CUSTOM_DATA = { foo: 'bar' };
+
+export function RTVIProvider({ children }: PropsWithChildren) {
+  const [client, setClient] = useState<RTVIClient | null>(null);
+
+  useEffect(() => {
+    console.log('Setting up Transport and Client');
+    const transport = new DailyTransport();
```

Comment:
Create `PipecatCloudTransport` would be nice, so we could use the same example to test all the available transports.